apps/vanilla/src/pages/exercise1/index.css

```
@@ -61,44 +66,75 @@ body {
   gap: 24px;
 }
 
+.content-container__books-container {
+  max-width: 900px;
+}
+
 .content-container__book-filters {
+  width: 320px;
   align-self: flex-start;
+  background: var(--surface-color);
+  outline: 1px solid var(--on-primary-color);
   position: sticky;
   top: 0;
-  background: var(--surface-color);
+}
+
+.content-container__book-group-title {
+  background-color: var(--surface-color);
   outline: 1px solid var(--on-primary-color);
+  position: sticky;
+  top: 0;
 }
 
-@media screen and (min-width: 1280px) {
+@media screen and (min-width: 786px) {
   .main-container {
     margin: 0 auto;
   }
 }
 
-@media screen and (max-width: 786px) {
+@media screen and (max-width: 1919px) {
```

```suggestion
@media (max-width: 1919px) {
```
If you don't have styles for `print`, then I see no reason to specify `screen`. 

Plus you can use the new syntax 
`@media (400px <= width <= 700px)`
Seems clearer to me)